django/core/handlers/base.py

```diff
@@ -34,7 +37,12 @@ def load_middleware(self):
         for middleware_path in reversed(settings.MIDDLEWARE):
             middleware = import_string(middleware_path)
             try:
-                mw_instance = middleware(handler)
+                if getattr(middleware, "_is_async", False):
```

Comment:
What about the following?

```suggestion
                if asyncio.iscoroutinefunction(getattr(middleware, '__call__', None)):
```